app/views/groups/new.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1 class="text-center mt-5">Add New Catagory</h1>
```

Comment:
- [ ] Based on the requirements,   please make sure when the user clicks on the "Back" button (<), the user navigates to the home page, there is currently no back button on your page 👍🏿

Reply:
done